dist/configs/config/map/default.cfg

```diff
@@ -1,3 +1,6 @@
+// This file is executed each time a new map is loaded.
+// In addition, a map-specific config is executed if present, e.g. config/map/plat23.cfg.
```

Comment:
Keep documentation about layout specific configs?

Reply:
I would report those lines (it can be rewritten if needed):

```
// it will load all the configs specified:
//   - map/default.cfg (for every map)
//   - map/plat23.cfg (for plat23, regardless of the layout)
//   - map/plat23/layout.cfg (for plat23 and layout "layout")
```

because this isn't clear enough to me (and does not mention layout):

```
// In addition, a map-specific config is executed if present, e.g. config/map/plat23.cfg.
```